core/shared/src/main/scala/io/udash/properties/PropertySeqCombinedReadableSeqProperty.scala

```diff
@@ -0,0 +1,37 @@
+package io.udash.properties
+
+import com.avsystem.commons.ISeq
+import io.udash.properties.Properties.{Patch, Property, ReadableProperty}
+import io.udash.properties.seq.AbstractReadableSeqProperty
+import io.udash.utils.Registration
+
+private[properties] final class PropertySeqCombinedReadableSeqProperty[A](value: ISeq[Property[A]])
+  extends AbstractReadableSeqProperty[A, ReadableProperty[A]] {
+
+  override val id: PropertyId = PropertyCreator.newID()
+  override protected[properties] val parent: ReadableProperty[_] = null
+
+  private val children = CrossCollections.createArray[ReadableProperty[A]]
+
+  value.foreach(property => {
+    children.+=(property.transform(value => value))
+    property.listen(_ => valueChanged())
```

Comment:
This implementation does not allow GC to clean up this property, when it's no longer needed, but the origin `property` is still in use. Take a look at `transform` implementation from `ReadableProperty`.